Assets/PlayroomKit/modules/MockMode/LocalPlayroomService.cs

```diff
@@ -167,17 +167,48 @@ public void ResetPlayersStates(string[] keysToExclude = null, Action onStatesRes
 
         public void CreateJoyStick(JoystickOptions options)
         {
-            throw new NotImplementedException();
+            DebugLogger.LogWarning("[MockMode] Joystick is currently not supported in local mode!");
         }
 
         public Dpad DpadJoystick()
         {
-            throw new NotImplementedException();
+            DebugLogger.LogWarning("[MockMode] Joystick is currently not supported in local mode!");
+            return default;
         }
 
         public void UnsubscribeOnQuit()
         {
-            throw new NotImplementedException();
+            DebugLogger.LogWarning("[MockMode] Unsubscribe on quit is currently not supported in local mode!");
         }
+
+        #region Turbased
+
+        public string GetChallengeId()
+        {
+            DebugLogger.LogWarning("[MockMode] Turn based API is currently not supported in local mode!");
```

Comment:
Note: We should create a table somewhere which contains what features are contained in which mock mode.